_docs/what-is-rh-open-innovation-labs.md

```diff
@@ -4,14 +4,44 @@ prev_section: index.html
 next_section: how-roil-works.html
 ---
 
-What is Red Hat Open Innovation Labs and why?
+What is Red Hat Open Innovation Labs, and why?
 =============================================
 
-Lorem ipsum dolor sit amet
---------------------------
+What is Red Hat Open Innovation Labs?
+-------------------------------------
+Red Hat Open Innovation Labs is a global team at Red Hat who work with customers, partners, and communities, helping them build innovative software and high-performing teams, in order to solve critical business challenges. And we have a good deal of fun doing it, too.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut id leo eu tortor vulputate mattis nec ac libero. Maecenas finibus est eu nisl egestas euismod. Nullam et finibus nulla. Nullam commodo non elit et imperdiet. Nullam tincidunt mollis egestas. Nulla nec lorem non turpis venenatis ornare. Donec id libero magna. Etiam sed ante massa.
+Red Hat Open Innovation Labs has come to mean many things:
+* Labs employees
+* Labs facilities
+* Labs ways of working
+* Labs offering portfolio
+* Labs push-button infrastructure and codebases
+* Labs residencies
 
-Nullam id lorem est. Interdum et malesuada fames ac ante ipsum primis in faucibus. In semper arcu enim, elementum feugiat diam porta quis. Aenean eget eros lacinia, lobortis lectus eu, faucibus nulla. Curabitur cursus purus ac mauris semper semper. Sed varius sagittis leo, sit amet vulputate nibh consequat quis. Etiam finibus pulvinar diam ac interdum. Cras molestie ante vel aliquam lacinia. Curabitur suscipit malesuada facilisis. Sed in lorem vitae orci molestie interdum. Nunc luctus eleifend libero eu vulputate.
+<!-- TODO: link to Labs Residency -->
+But the easiest way to get to understand Labs is by learning more about our flagship offering: the Labs Residency.
 
-> Cras molestie ante vel aliquam lacinia. Curabitur suscipit malesuada facilisis. Sed in lorem vitae orci molestie interdum. Nunc luctus eleifend libero eu vulputate.
\ No newline at end of file
+
+A brief history of Labs
+---------------------------------
+Red Hat Open Innovation Labs was founded in 2016. At that time, Red Hat had established a compelling platform for hybrid cloud computing, topped off with the recent release of a kubernetes-based OpenShift container platform. To be successful in the marketplace, it was clear that Red Hat needed to help their clients, partners, and communities put that cloud computing platform to use. And that meant getting applications on the platform that provided signifigant additional business value. That value might equate to designing and deploying a new revenue-generating product to pursue a new market. Or it might mean developing greater business agility, or improved operational efficiencies, by leveraging the cloud. We wanted (and still want) to use open source technologies to change the world for the better, and become the defining technology company of the 21st century.
```

Comment:
```suggestion
Red Hat Open Innovation Labs was founded in 2016. At that time, Red Hat had established a compelling platform for hybrid cloud computing, topped off with the recent release of a kubernetes-based OpenShift container platform. To be successful in the marketplace, it was clear that Red Hat needed to help their clients, partners, and communities put that cloud computing platform to use. And that meant getting applications on the platform that provided signifigant additional business value. That value might equate to designing and deploying a new revenue-generating product to pursue a new market. Or it might mean developing greater business agility, or improved operational efficiencies, by leveraging the cloud. We wanted (and still want) to use open source technologies, principles, and practices to change the world for the better, and become the defining technology company of the 21st century.
```